L79-cpp-async-response-reader-destruction.md

```diff
@@ -0,0 +1,121 @@
+API changes for `ClientAsyncResponseReaderInterface` destruction
+----
+* Author: jacobsa
+* Approver: soheilhy
+* Status: Draft
+* Implemented in: C++
+* Last updated: 2021-05-14
+* Discussion at: <google group thread> (filled after thread exists)
```

Comment:
Before merging the PR, we usually send an email to the mailing list and then merge this PR.

Reply:
Thanks. Do you want me to start the thread? Can you tell me how/give me an example?

Reply:
Started the thread and left a link.